postfix/datadog_checks/postfix/data/conf.yaml.example

```diff
@@ -57,3 +57,21 @@ instances:
     tags:
       - optional_tag3
       - optional_tag4
+
+
+## Log Section (Available for Agent >=6.0)
+##
+## type - mandatory - Type of log input source (tcp / udp / file / windows_event)
+## port / path / channel - mandatory - Set port if type is tcp or udp. Set path if type is file and channel if windows_event
+## service - mandatory - Name of the service owning the log
```

Comment:
```suggestion
## service - mandatory - Name of the service that owns the log
```

Is that accurate? Or is it the name of the service that generates the logs, for example?

Reply:
it's the name of the service that generates the logs not own it, since we want to bind those logs to the corresponding APM service.

Reply:
Ok, got it. Let's be clear about that.
```suggestion
## service - mandatory - Name of the service that generated the log
```